src/js/rtc_peer_connection_factory.js

```diff
@@ -1,21 +1,26 @@
-import {assertTrue, getChromeBrowserVersion, hitch, isChromeBrowser, isFunction} from './utils';
+import {assertTrue, hitch, isFunction} from './utils';
 import {
-    CHROME_SUPPORTED_VERSION,
     DEFAULT_ICE_CANDIDATE_POOL_SIZE,
     NETWORK_CONNECTIVITY_CHECK_INTERVAL_MS,
     RTC_PEER_CONNECTION_CONFIG,
     RTC_PEER_CONNECTION_IDLE_TIMEOUT_MS,
     RTC_PEER_CONNECTION_OPTIONAL_CONFIG,
     SOFTPHONE_ROUTE_KEY
 } from './rtc_const'
+import CCPInitiationStrategyInterface from "./strategies/CCPInitiationStrategyInterface";
```

Comment:
CCPInitiationStrategyInterface - Is the naming correct? Would SoftphoneManagerInitiationStrategyInterface make more sense?

Reply:
Yeah, we have reviewed the naming internally.